packages/contracts/src/ArbitrumVerifier.sol

```diff
@@ -54,17 +54,17 @@ contract ArbVerifier is IEVMVerifier {
             abi.decode(proof, (ArbWitnessData, StateProof));
 
         //Get the node from the rollup contract
-        Node memory rblock = rollup.getNode(arbData.nodeIndex);
+        // Node memory rblock = rollup.getNode(arbData.nodeIndex);
 
         //The confirm data is the keccak256 hash of the block hash and the send root. It is used to verify that the rblock is a subject of the layer 2 block that is being proven.
-        bytes32 confirmData = keccak256(
-            abi.encodePacked(
-                keccak256(arbData.rlpEncodedBlock), arbData.sendRoot
-            )
-        );
+        // bytes32 confirmData = keccak256(
```

Comment:
Create an issue to investigate that!

Reply:
https://github.com/orgs/blockful-io/projects/6?pane=issue&itemId=81962375